openapi3/swagger_loader.go

```diff
@@ -567,3 +577,85 @@ func (swaggerLoader *SwaggerLoader) resolveExampleRef(swagger *Swagger, componen
 	}
 	return nil
 }
+
+func (swaggerLoader *SwaggerLoader) resolvePathItemRef(swagger *Swagger, endpoint string, ref string) error {
+	if !swaggerLoader.IsExternalRefsAllowed {
```

Comment:
Are you not able to use `resolveComponent`? See for instance how `resolveRequestBodyRef` uses it.

Reply:
I am already check the resolveComponent function, i think it just resolve the network fille.
But doesnt resolve the relative path file, is it right ?

Reply:
Looking at the code, `resolveComponent` calls `LoadSwaggerFromURI` which [looks like it falls back to resolving `file://` URLs](https://github.com/getkin/kin-openapi/blob/d3180292eead101a815a9291fd8b6aca8b45927e/openapi3/swagger_loader.go#L67). If that is not possible then it should be made more explicit, but I believe constructing a `url.URL` with a the proper `.Path` set should arrive there. Let's try using `resolveComponent` :)